packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -96,19 +96,43 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
                   pagePhase == PagePhase.linksLoading
                       ? 'Loading deep links...'
                       : 'Validating deep links...',
-                  style: Theme.of(context).subtleTextStyle,
+                  style: theme.subtleTextStyle,
                 ),
               ],
             );
           case PagePhase.linksValidated:
             return const _ValidatedDeepLinksView();
           case PagePhase.noLinks:
             // TODO(hangyujin): This is just a place holder to add UI.
-            return const Text('Your flutter project has no Links to verify.');
+            return const CenteredMessage(
+              'Your Flutter project has no Links to verify.',
+            );
+          case PagePhase.analyzeErrorPage:
+            assert(controller.currentAppLinkSettings?.error != null);
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                const Text(
+                  'Failed to retrieve deep links from the Flutter project. '
+                  'This can be a result of errors in the project.',
+                ),
+                const SizedBox(height: densePadding),
+                Expanded(
+                  child: OutlineDecoration(
```

Comment:
nit: I recommend removing this outline. Looking at the screenshot, this creates a double border at the bottom and clashes a bit with the surrounding rounded outline of the container.

I also recommend wrapping the `SingleChildScrollView` in a `Scrollbar` widget with `thumbVisibility` set to `true` so that it is more obvious to the user that this is a scrollable view if the content extends the view bounds.